tests/unit/test_policy_manager.py

```diff
@@ -156,22 +100,26 @@ def test_policy_is_tagged_with_expiry_time() -> None:
     )
 
     mock_client.create_policy.assert_called_once()
-    print(mock_client.create_policy.mock_calls)
     assert mock_client.create_policy.call_args.kwargs["tags"] == [
         {"Key": "Product", "Value": "grant-user-access"},
         {"Key": "Expires_At", "Value": "2012-01-15T00:00:01Z"},
     ]
 
 
+def _get_policy(policy_arn: str) -> Dict[str, Any]:
+    return resp.POLICIES_MAP[policy_arn]
+
+
 @mock_iam  # type: ignore
 def test_find_expired_policies_returns_arns_of_no_longer_needed_policies() -> None:
-    # using a hand rolled mock here as moto does not return back policy tags
-    mock_client = Mock(list_policies=Mock(return_value=LIST_POLICIES))
+    mock_client = Mock(
+        list_policies=Mock(return_value=resp.LIST_POLICIES),
+        get_policy=Mock(side_effect=_get_policy),
+    )
 
     expired = PolicyCreator(mock_client).find_expired_policies(
         current_time=datetime(year=2021, month=1, day=1, hour=1, minute=1, second=1)
     )
-
     mock_client.list_policies.assert_called_once_with(path_prefix="/Lambda/GrantUserAccess/")
```

Comment:
```suggestion
     mock_client.list_policies.assert_called_once_with(path_prefix="/Lambda/GrantUserAccess/")
     mock_client.get_policy.assert_has_calls()
```
Might be good to check our new endpoint is called in the way we expect